pypass/tests/test_passwordstore.py

```diff
@@ -53,7 +60,9 @@ def tearDown(self):
     def test_constructor(self):
         # Construct on properly initialized directory
         store = PasswordStore(self.dir)
-        self.assertEqual(store.gpg_id, '5C5833E3')
+        self.assertEqual(store._get_gpg_id(self.dir), '5C5833E3')
+        email_path = os.path.join(self.dir, 'Email')
```

Comment:
that would go somewhere else than test_constructor since we are testing _get_gpg_id and not the constructor

Reply:
I decided to remove this because we're only testing if we successfully used Python's .write in the setUp, a bit silly yeah.